tests/e2e/mapper/configuration/config.yaml

```diff
@@ -0,0 +1,20 @@
+mqtt:
```

Comment:
since we will have modbus mapper e2e as well , we should have one more folder for bluetooth and then place all configuration under it. like
```sh 
tests/e2e/mapper/bluetooth/configuration
```

Reply:
+1

Reply:
Updated